pkg/app/mysql.go

```diff
@@ -0,0 +1,244 @@
+package app
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+	awsconfig "github.com/kanisterio/kanister/pkg/config/aws"
+	"github.com/kanisterio/kanister/pkg/format"
+	"github.com/kanisterio/kanister/pkg/kube"
+	"github.com/kanisterio/kanister/pkg/testutil"
+	log "github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+type chartInfo struct {
+	helmrepoURL   string
+	helmChartName string
+	helmAppname   string
+}
+
+type MysqlDB struct {
+	cli       kubernetes.Interface
+	namespace string
+	password  string
+	accessID  string
+	secretKey string
+	region    string
+	chart     chartInfo
+}
+
+func NewMysqlDB(helmrepoURL, helmChart, helmAppname string) App {
+	return &MysqlDB{
+		password: "mysecretpassword",
+		chart: chartInfo{
+			helmrepoURL:   helmrepoURL,
+			helmChartName: helmChart,
+			helmAppname:   helmAppname,
+		},
+	}
+}
+
+func (mdb *MysqlDB) Init(ctx context.Context) error {
+	var ok bool
+	mdb.region, ok = os.LookupEnv(awsconfig.Region)
```

Comment:
Do these need to be set in the mysql struct? Can they instead be handled by the profile?

Reply:
Hi @tdmanv 
I agree, and it was duplicated here in mysql struct. These values are closely related to the profile and should be set there only.
I will make the respective changes.